### PR TITLE
Add E2E tests for authorisation adjustment

### PR DESCRIPTION
**Pull request by Kwok-he-Chu**

### CODE REVIEW

File: tests/authorisation-adjustment/card.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    // Find iframe and fill "CVC / CVV" field
+    const cvcFrame = await page.frameLocator('iframe[title*="security code"]');
+    await cvcFrame.getByPlaceholder('3 digits').fill('737');
+   
+    // Find and fill "Name on card" field - Note: this field is not contained within an iframe
+    await page.getByPlaceholder('J. Smith').fill('J. Smith');
+
+    // Click "Pay" button
+    const payButton = page.locator('.adyen-checkout__button__text >> visible=true');
```
Comment (ayodejidev) [inline tests/authorisation-adjustment/card.spec.js:37]: Nice work here Kwok!

Suggestion:
Can we handle the case where the payment is not successful? Like how long (timeout) should apply?

File: tests/authorisation-adjustment/admin-panel.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// @ts-check
```
Comment (ayodejidev) [inline tests/authorisation-adjustment/admin-panel.spec.js:1]: Suggestion:

Can we have comments here as well so its clear what case is tested. e.g // Verify page title

Comment (Kwok-he-Chu) [reply tests/authorisation-adjustment/admin-panel.spec.js:1]: This test is not doing much except that it verifies that the admin panel is there. To-be-fair we can leave it out too
We'll add the E2E logic once we pick-up https://github.com/adyen-examples/adyen-testing-suite/issues/24

File: tests/authorisation-adjustment/card.spec.js (hunk 1 above)
Comment (Kwok-he-Chu) [reply tests/authorisation-adjustment/card.spec.js:37]: Will do

**Pull request merged**